app/views/search/_collection_details.html.erb

```diff
@@ -94,16 +94,25 @@
               <!-- /ko -->
             </dd>
           </dl>
+          <!-- end of second row -->
+
+          <ul>
+            <!-- ko foreach: $data.highlighted_urls -->
+            <li><a data-bind="attr: {href: $data.url}, text: $data.name" target="_blank"></a></li>
+            <!-- /ko -->
+            <!-- ko if: $data.related_urls.length > 0 -->
```

Comment:
I happened to read a stackoverflow Q/A that ```<!-- ko``` can be used for 'virtual elements'. But it doesn't say that it is a convention to move that into the data-bind attr. It is just my interpretation :)